prow/config/jobs/test-infra.yaml

```diff
@@ -34,6 +34,15 @@ jobs:
   - name: deploy-prow
     types: [postsubmit]
     regex: '^prow/cluster/(?:gcsweb/|velodrome/|monitoring/|build/|private/|[^/]+\.yaml$)'
+    reporter_config:
+      slack:
+        channel: "oncall"
```

Comment:
doesn't every pustsubmit already report to slack? is this just so it goes to oncall instead of test-failures?

Reply:
yeah they report on faillures and the reports are mixed with all other jobs https://github.com/istio/test-infra/blob/c0a883877ac28436cf3b2baaebd9b474d8123197/prow/config.yaml#L109. This is for replacing the manual post from oncall, so that we have one less thing to do on Mondays